src/nvim/eval.lua

```diff
@@ -9955,7 +9955,11 @@ M.funcs = {
 
     ]=],
     name = 'setqflist',
-    params = { { 'list', 'any[]' }, { 'action', 'string' }, { 'what', 'table' } },
+    params = {
+      { 'list', 'vim.quickfix.entry[]' },
```

Comment:
I don't think so. I'm not sure what problem that solves. There is a possibility that these types never change; If we add `__nvim_types` then it is more likely they will change. These types are only used for LSP and linting, so they are low risk.

Reply:
I mean if plugin authors start referencing the types defined in our meta files, then changing those types will "break" those consumers.

Reply:
It's not been a problem for me and my plugins. It doesn't break the plugins, it just causes some diagnostics which are trivial to resolve.

Communicating to me that using these types _might_ change doesn't improve anything for me as a plugin author. I'm going to use them regardless, because the alternatives are much worse.